polkadot/node/core/approval-voting/src/tests.rs

```diff
@@ -777,12 +778,7 @@ async fn import_block(
 	overseer_send(
 		overseer,
 		FromOrchestra::Signal(OverseerSignal::ActiveLeaves(ActiveLeavesUpdate::start_work(
-			ActivatedLeaf {
-				hash: *new_head,
-				number,
-				status: LeafStatus::Fresh,
-				span: Arc::new(jaeger::Span::Disabled),
-			},
+			fresh_leaf(*new_head, number),
```

Comment:
Not at the side: We found this fresh/stale mechanism to be redundant. We should remove it at [some point](https://github.com/paritytech/polkadot-sdk/issues/768).

Reply:
I've renamed it to new_leaf to account for this issue being worked on. Also with this helper function, we won't have to modify tons of files to implement this issue.